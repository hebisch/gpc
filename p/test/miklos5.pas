program testit;

var x:real;

begin
 x:=355/113;
 writeln(round(355/113):10);
 writeln(355/113:12:8);
 writeln(round(355/113):10,' ',355/113:12:8);
 writeln(355/113:12:8,' ',round(355/113));
 writeln(round(355/113*100)); writeln(round(355/113*10000));
 writeln(round(x):10);
 writeln(x:12:8);
 writeln(round(x),' ',x:12:8);
 writeln(round(x*100)); writeln(round(x*10000));
end.
