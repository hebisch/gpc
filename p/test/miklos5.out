         3
  3.14159292
         3   3.14159292
  3.14159292 3
314
31416
         3
  3.14159292
3   3.14159292
314
31416
